Sources/MapperParse.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  MapperParse.swift
+//  ObjectMapper-iOS
+//
+//  Created by yixiaojichunqiu on 2019/6/13.
+//  Copyright © 2019 hearst. All rights reserved.
+//
+
+import Foundation
+
+public final class MapperParse
+{
+	/// Converts an Object to a JSON string with option of pretty formatting
+	public static func toJSONString(_ JSONObject: Any, prettyPrint: Bool) -> String? {
+		let options: JSONSerialization.WritingOptions = prettyPrint ? .prettyPrinted : []
+		if let JSON = MapperParse.toJSONData(JSONObject, options: options) {
+			return String(data: JSON, encoding: String.Encoding.utf8)
```

Comment:
Please pass just `.utf8` instead of `String.Encoding.utf8`

 `return String(data: JSON, encoding: .utf8)`